requirements.txt

```diff
@@ -4,6 +4,7 @@ numpy
 numpydoc
 ophyd
 pcdsutils
+PyQt5
```

Comment:
`dev_requirements` was unchanged here, commenting on this file as it's closest. Before I forget:

`dev_requirements.txt`:
* `doctr` needs to be removed
* `doctr-verisons-menu` needs to be removed

`docs_requirements.txt` needs to be added with:
* sphinx and the other docs-building requirements
* `docs-versions-menu`
*